packages/utilities/psammead-styles/src/font-families-config.js

```diff
@@ -0,0 +1,9 @@
+const helmet = 'Helmet, Freesans, Helvetica, Arial, sans serif';
```

Comment:
Am I missing something? What happened to ReithSerif?

Reply:
Am looking at two ways of approaching this. 
1. We have `latin` return and object with `sans` and `serif`. But then that would be inconsistent because other scripts would simply return a string.
2. We add `script` names `latinSans` and `latinSerif` and then later on in the simorgh service config add a script name key which will have an array of scripts.